### PR TITLE
Update chain.json for archway mainnet

### PR DESCRIPTION
**Pull request by lufa23**
- Updated the chain.json

- Some rpc endpoints had been removed as they were not active

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Please see the comments from this PR: https://github.com/cosmos/chain-registry/pull/2456

**Comment by jlehtimaki** [comment]
> Please see the comments from this PR: #2456

@JeremyParish69 
Hey, we have seen the comments and addressed those ones which we think is valid in this PR.
We don't want to have untested endpoints so we removed the ones that were not working.
We will add more versions and compatibility on the next releases and when there's a need for it.
 
 Thank you.

**Comment by JeremyParish69** [comment]
> > Please see the comments from this PR: #2456
> 
> @JeremyParish69 Hey, we have seen the comments and addressed those ones which we think is valid in this PR. We don't want to have untested endpoints so we removed the ones that were not working. We will add more versions and compatibility on the next releases and when there's a need for it.
> 
> Thank you.

Thanks for the response, but there are many other comments which are all valid. I'd have the make the same comments here on this PR

File: archway/assetlist.json [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
-            "exponent": 18
-          }
-        ],
-        "base": "aarch",
-        "name": "Archway",
-        "display": "archway",
-        "symbol": "ARCH",
-        "logo_URIs": {
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "archway-1",
```
Comment (JeremyParish69) [inline archway/assetlist.json:3]: chain_name is 'archway'

File: archway/chain.json [hunk 2]
```diff
@@ -29,42 +29,34 @@
...
+    "cosmwasm_enabled": true,
+    "ibc_go_version": "v4.3.1",
     "genesis": {
-      "genesis_url": "https://github.com/archway-network/networks/blob/main/archway-1/genesis/genesis.json.gz"
+      "genesis_url": "https://github.com/archway-network/networks/raw/main/archway-1/genesis/genesis.json.gz"
     },
     "versions": [
       {
-        "name": "genesis",
-        "recommended_version": "v1.0.0",
```
Comment (JeremyParish69) [inline archway/chain.json:55]: Can you please re-add this stuff (from recommended_version to binaries) back in please?

Comment (JeremyParish69) [reply archway/chain.json:55]: FYI I added it in

**Pull request merged**